meteor/sofie-server/data-access/repositories/mongo/mongo-rundown-playlist-repository.ts

```diff
@@ -0,0 +1,72 @@
+import { RundownRepository } from '../interfaces/rundown-repository'
+import { BaseMongoRepository } from './base-mongo-repository'
+import { Rundown } from '../../../model/entities/rundown'
+import { MongoDatabase } from './mongo-database'
+import { MongoEntityConverter, MongoRundownPlaylist } from './mongo-entity-converter'
+import { BasicRundown } from '../../../model/entities/basic-rundown'
+
+const RUNDOWN_PLAYLIST_COLLECTION_NAME: string = 'rundownPlaylists'
+
+export class MongoRundownPlaylistRepository extends BaseMongoRepository implements RundownRepository {
+	constructor(
+		mongoDatabase: MongoDatabase,
+		mongoEntityConverter: MongoEntityConverter,
+		private rundownRepository: RundownRepository
+	) {
+		super(mongoDatabase, mongoEntityConverter)
+	}
+
+	protected getCollectionName(): string {
+		return RUNDOWN_PLAYLIST_COLLECTION_NAME
+	}
+
+	async getBasicRundowns(): Promise<BasicRundown[]> {
+		this.assertDatabaseConnection('getRundownPlaylists')
+		const rundowns: BasicRundown[] = await this.rundownRepository.getBasicRundowns()
```

Comment:
Since you are fetching BasicRundowns and not Rundowns a more apt name for your variable would be `basicRundowns`

Reply:
Very true, i have changed it.